.gitignore

```diff
@@ -0,0 +1,4 @@
+vendor
+.phpunit.result.cache
+.idea
+src/phars/index.html
```

Comment:
As a rule of thumb, I'd ask you to end every file on a new line character (here and elsewhere).

Reply:
`.gitattributes` 👍🏻 

As for an `.editorconfig`, I have no objections against it, though I also don't see much use for it. Fine if others find it useful.

Reply:
I have added `.gitattributes` but did not add an `.editorconfig`. I agree there's not much use for it if it's not strictly enforced.